apps/eclipse/eclipse.talon

```diff
@@ -4,7 +4,6 @@ app: eclipse
 tag(): user.find_and_replace
 tag(): user.line_commands
 # tag(): user.multiple_cursors
-# tag(): user.snippets
```

Comment:
I don't think we should hide snippets behind a tag anymore since they will be globally available

Reply:
agreed